### PR TITLE
Tooltips and icon change 

### PR DESCRIPTION
**Pull request by Pajsen9263**
I added tooltips as suggested by ticket https://github.com/digitalungdom-se/digitalungdom.se/issues/158 and I changed the forum icon in the footer to the discord logo as suggested by ticket https://github.com/digitalungdom-se/digitalungdom.se/issues/155

### CODE REVIEW

**Review by Nautman — CHANGES_REQUESTED** [review]
Great PR! Please fix the margin, and afterwards you have my approval to do a merge. If you want to, you can do a squash merge by clicking the dropdown on the merge button.

EDIT:
Small tips for the future :)
1. Don't write "I" in a commit.
2. Use GitHub Desktop (or other git tool) to keep track of changes and not forget to push changes from some files.

File: src/features/Footer.tsx
Comment (Pajsen9263) [reply src/features/Footer.tsx]: I have fixed that now! 

image

**Review by Nautman — APPROVED** [review]
Looks great!

**Pull request merged**
